File: src/main/java/de/moritzf/quota/idea/ui/toolbar/CustomComponentStatusBarWidget.java
package de.moritzf.quota.idea.ui.toolbar;

import com.intellij.openapi.wm.CustomStatusBarWidget;
import com.intellij.openapi.wm.StatusBarWidget;
import org.jetbrains.annotations.Nullable;

public abstract class CustomComponentStatusBarWidget implements CustomStatusBarWidget {
    @Override
    public @Nullable StatusBarWidget.WidgetPresentation getPresentation() {
        return null;
    }
}

File: src/main/kotlin/de/moritzf/quota/idea/ui/toolbar/CustomComponentStatusBarWidget.kt
package de.moritzf.quota.idea.ui.toolbar

import com.intellij.openapi.wm.CustomStatusBarWidget
import com.intellij.openapi.wm.StatusBarWidget
import org.jetbrains.annotations.Nullable

abstract class CustomComponentStatusBarWidget : CustomStatusBarWidget {
    @Nullable
    override fun getPresentation(): StatusBarWidget.WidgetPresentation? = null
}
